src/components/BoardLIst/BoardsList.js

```diff
@@ -23,6 +23,7 @@ const BoardsList = ({ boards, createBoardBtnClick, onBoardRemoved }) => {
     const [selectedBoard, setSelectedBoard] = useState(null)
     const editedTitleRef = useRef(null)
     const editedDescriptionRef = useRef(null)
+    const { userId } = useAuth()
 
     const openPopUp = (e) => {

```

Comment:
```js
const handleEditBtnClick = () => {
  editBordBtnClick(selectedBoard.id)
}
```

the same for delete board

then call handleEditBtnClick instead of openPopUp